composer.json

```diff
@@ -22,19 +22,30 @@
     },
     "require-dev": {
         "yiisoft/yii2-redis": "*",
-        "php-amqplib/php-amqplib": "*",
+        "php-amqplib/php-amqplib": "2.7.x-dev",
         "pda/pheanstalk": "*",
         "jeremeamia/superclosure": "*",
         "yiisoft/yii2-debug": "*",
         "yiisoft/yii2-gii": "*",
-        "phpunit/phpunit": "~4.4"
+        "phpunit/phpunit": "~4.4",
+        "enqueue/amqp-lib": "0.7.x-dev"
     },
     "suggest": {
         "ext-pcntl": "Need for process signals.",
         "yiisoft/yii2-redis": "Need for Redis queue.",
         "pda/pheanstalk": "Need for Beanstalk queue.",
         "php-amqplib/php-amqplib": "Need for AMQP queue.",
-        "ext-gearman": "Need for Gearman queue."
+        "ext-gearman": "Need for Gearman queue.",
+        "php-enqueue/amqp-ext": "Needs for support of AMQP with queue interop",
```

Comment:
Needs for support of AMQP with queue interop → Required for AMQP with queue interop